src/Tools/MicroBuild/cibuild.ps1

```diff
@@ -22,7 +22,11 @@ try {
     Exec { & $nuget locals all -clear }
 
     $msbuild = Ensure-MSBuild
-    & $msbuild /nodereuse:false /p:Configuration=Release /p:SkipTest=true Build.proj 
+
+    # The /nowarn exception can be removed once we fix the following:
+    #  - 3277: https://github.com/dotnet/roslyn/issues/17325
+    #  - 4011: https://github.com/dotnet/roslyn/issues/18151
+    & $msbuild /nodereuse:false /p:Configuration=Release /p:SkipTest=true Build.proj /warnaserror /nowarn:3277,4011
```

Comment:
This doesn't break actual microbuild? Can you confirm we're on the Dev15 pool at this point before merging?